template/README.md

```diff
@@ -18,7 +18,7 @@ repository.
 This template was originally forked from the
 [Substrate Node Template](https://github.com/substrate-developer-hub/substrate-node-template). You
 can find more information on features on this template there, and more detailed usage on the
-[Substrate Developer Hub Tutorials](https://docs.substrate.io/tutorials/v3/) that use this heavily.
+[Substrate Developer Hub Tutorials](https://docs.substrate.io/tutorials/integrate-with-tools/access-evm-accounts/) that use this heavily.
```

Comment:
This template began as a fork of the [Substrate Node Template](https://github.com/substrate-developer-hub/substrate-node-template). For a deeper understanding of the features provided by this template, as well as additional guidance, you can refer to the following resources:

- [Substrate Developer Hub Tutorials](https://docs.substrate.io/tutorials/)
- [Tutorial on Accessing EVM Accounts](https://docs.substrate.io/tutorials/integrate-with-tools/access-evm-accounts/)

Would you like to update this phase in this way? Prefer to contain both the substrate and evm account tutorials.